db_schema/queries/v1/reportApplePlays.sql

```diff
@@ -1,8 +1,8 @@
 SELECT
```

Comment:
Hum, `COUNT(DISTINCT())` wouldn't work since we can't identify the unique listeners, how about we take the `MAX` for the given time period as a workaround? Not 100% accurate, but better than the sum.

```sql
SELECT
  account_id as podcast_id,
  SUM(atl_playscount) as apple_playscount,
  SUM(atl_totaltimelistened) as apple_totaltimelistened,
  MAX(atl_uniqueengagedlistenerscount) as apple_uniqueengagedlistenerscount,
  MAX(atl_uniquelistenerscount) as apple_uniquelistenerscount
FROM
  appleTrendsPodcastListeners
WHERE
  atl_date >= @start
  AND atl_date <= @end
  AND account_id = @podcast_id
GROUP BY
  account_id;
```

Reply:
they only way to get this value is to ask spotify/apple for the aggregated data. everything else is off.